Intersect.Server.Core/Maps/MapInstance.cs

```diff
@@ -713,26 +715,31 @@ private void AddItem(MapItem item)
         }
 
         TileItems[item.TileIndex]?.TryAdd(item.UniqueId, item);
+        
+        //TODO: Invoke the event for the item being added to the map, maybe: 
+        // _mapHelper?.ItemAdded?.Invoke(this, new ItemEventArgs(source, item));
     }
 
     /// <summary>
     /// Spawn an item to this map instance.
     /// </summary>
+    /// <param name="source">The source of the item, which can be an entity or player</param>
```

Comment:
Description should be `The source of the item, e.g. a player who dropped it, or a monster who spawned it on death` and if we add map sources it should be `The source of the item, e.g. a player who dropped it, or a monster who spawned it on death, or the map instance in which it was spawned`